executor.go

```diff
@@ -379,6 +379,19 @@ func (e *executor) runJob(j internalJob, jIn jobIn) {
 	}
 }
 
+func (e *executor) callJobWithRecover(j internalJob) (err error) {
+	defer func() {
+		if recoverData := recover(); recoverData != nil {
+			_ = callJobFuncWithParams(j.afterJobRunsWithPanic, j.id, j.name, recoverData)
+
+			// if panic is occurred, we should return an error
+			err = ErrPanicRecovered
```

Comment:
Is this error ever readable by the caller? Seems ineffectual. But I could be wrong 😁

Reply:
The caller can catch this error via an EventListener

```go
WithEventListeners(
  AfterJobRunsWithError(
    func(jobID uuid.UUID, jobName string, err error) {
      if errors.Is(err, gocron.ErrPanicRecovered) {
         // ....
      }
    },
  ),
),
```